src/include/sof/audio/pipeline.h

```diff
@@ -42,6 +42,13 @@
 #include <sof/schedule.h>
 #include <uapi/ipc/topology.h>
 
+/*
+ * This flag disables firmware-side xrun recovery.
+ * It should remain enabled in the situation when the
+ * recovery is delegated to the outside of firmware.
+ */
+#define NO_XRUN_RECOVERY 1
```

Comment:
@slawblauciak this part will need to be updated when @jajanusz has merged the build system updates as an a build menu option, but it can be a seperate PR later on.

Reply:
Yeah, I was thinking just that